stl/src/locale0.cpp

```diff
@@ -146,7 +146,7 @@ _MRTIMP2_PURE locale __CLRCALL_PURE_OR_CDECL locale::empty() { // make empty tra
 }
 
 _MRTIMP2_PURE locale::_Locimp* __CLRCALL_PURE_OR_CDECL locale::_Init(bool _Do_incref) { // setup global and "C" locales
-    locale::_Locimp* ptr = nullptr;
+    locale::_Locimp* ptr;

```

Comment:
Presumably, they did that because the `ptr` variable is immediately initialized to a different value after the `_BEGIN_LOCK(_LOCK_LOCALE)` macro, so the initialization to `nullptr` is ultimately redundant. I do like to believe that any optimizing compiler with half of a brain would remove that assignment anyways, though.

(On another note, I *really* don't like those `_BEGIN_LOCK` and `_END_LOCK` macros. I mean, `_END_LOCK` is literally defined as just an end brace character, for crying out loud! Perhaps a separate PR should be make just to remove those macros, unless there's some really strange reason for its continued existence.)